{"version":"3.8.0","messages":["\nAn error occurred while loading ./spec/ruby_basics_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0m\u001b[32mdef\u001b[0m \u001b[1;34margue\u001b[0m(a)\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mreturn\u001b[0m a\u001b[0m\n\u001b[31m  \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mArgumentError:\u001b[0m\n\u001b[31m  wrong number of arguments (given 0, expected 1)\u001b[0m\n\u001b[36m# ./lib/ruby_basics.rb:13:in `argue'\u001b[0m\n\u001b[36m# ./lib/ruby_basics.rb:17:in `<top (required)>'\u001b[0m\n\u001b[36m# ./spec/ruby_basics_spec.rb:2:in `require_relative'\u001b[0m\n\u001b[36m# ./spec/ruby_basics_spec.rb:2:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000423119,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":1},"summary_line":"0 examples, 0 failures, 1 error occurred outside of examples"}